bazel/repository_locations.bzl

```diff
@@ -849,8 +889,8 @@ DEPENDENCY_REPOSITORIES_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ SDK)",
         project_desc = "WebAssembly for Proxies (C++ SDK)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-sdk",
-        version = "7afb39d868a973caa6216a535c24e37fb666b6f3",
-        sha256 = "213d0b441bcc3df2c87933b24a593b5fd482fa8f4db158b707c60005b9e70040",
+        version = "b5ff880ed13d93dde9faf53d0aa05b216cc60b86",
```

Comment:
I think this is breaking. I just tested, extension compiled with 2.0.x emscripten tool chain does not work with 1.8.0.

Reply:
Elaborate? That shouldn't be the case.

Reply:
The extension compiled with proxy wasm sdk master could only function with master proxy, but not 1.8 proxy. I assume it is because of the toolchain, but it might be because of the missing js glue code.

Reply:
Also, keep in mind that the Emscripten update in this PR only affects tests, not the host environment.

Reply:
> The extension compiled with proxy wasm sdk master could only function with master proxy, but not 1.8 proxy. I assume it is because of the toolchain, but it might be because of the missing js glue code.

But that's unrelated to this PR. What you need to test for this PR is that extensions that shipped with 1.8.0 still work with Istio Proxy build from this PR.

Reply:
I think the issue is that we shipped Envoy in Istio Proxy 1.8.0 without proxy-wasm/proxy-wasm-cpp-host#72, so it doesn't know about Wasm Reactors and it cannot load modules built new Emscripten.

Reply:
> But that's unrelated to this PR. What you need to test for this PR is that extensions that shipped with 1.8.0 still work with Istio Proxy build from this PR.

I am thinking about user who uses Envoy wasm build rule might get a break because of the tool chain upgrade. And yeah glad you find the root cause! but I am still not so confident on updating so many dependencies on a patch branch..

Reply:
Same comment as https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/102#issuecomment-730714090. I am planning to create a istio-release/1.8 branch in proxy-wasm-cpp-sdk which excludes the emscripten update. Can we update sha to that once that is merged?

Reply:
Sure, do you want to do it in this PR?

Reply:
Actually, it does not matter. Envoy modules could be built with its own rules. I will do the cherry-pick to sdk v1.8 branch, but we can leave the sha as is here.